File: keyboards.py
from aiogram.types import InlineKeyboardMarkup, InlineKeyboardButton


def lang_kb() -> InlineKeyboardMarkup:
    return InlineKeyboardMarkup(inline_keyboard=[[
        InlineKeyboardButton(text="🇷🇺 Русский", callback_data="lang:ru"),
        InlineKeyboardButton(text="🇩🇪 Deutsch", callback_data="lang:de"),
    ]])


def length_kb() -> InlineKeyboardMarkup:
    return InlineKeyboardMarkup(inline_keyboard=[
        [
            InlineKeyboardButton(text="1000", callback_data="len:1000"),
            InlineKeyboardButton(text="2000", callback_data="len:2000"),
        ],
        [
            InlineKeyboardButton(text="3000", callback_data="len:3000"),
            InlineKeyboardButton(text="5000", callback_data="len:5000"),
        ],
    ])


def output_kb() -> InlineKeyboardMarkup:
    return InlineKeyboardMarkup(inline_keyboard=[
        [
            InlineKeyboardButton(text="📝 WordPress", callback_data="out:wp"),
            InlineKeyboardButton(text="📢 Telegram", callback_data="out:tg"),
        ],
        [InlineKeyboardButton(text="📋 Скопировать", callback_data="out:copy")],
        [InlineKeyboardButton(text="🔄 Новая статья", callback_data="restart")],
    ])

File: src/lib/keyboards.ts
import { Markup } from 'telegraf'

export function langKeyboard() {
  return Markup.inlineKeyboard([
    [
      Markup.button.callback('🇷🇺 Русский', 'lang:ru'),
      Markup.button.callback('🇩🇪 Deutsch', 'lang:de'),
    ],
  ])
}

export function lengthKeyboard() {
  return Markup.inlineKeyboard([
    [
      Markup.button.callback('1 000', 'len:1000'),
      Markup.button.callback('2 000', 'len:2000'),
    ],
    [
      Markup.button.callback('3 000', 'len:3000'),
      Markup.button.callback('5 000', 'len:5000'),
    ],
  ])
}

export function outputKeyboard() {
  return Markup.inlineKeyboard([
    [
      Markup.button.callback('📝 WordPress', 'out:wp'),
      Markup.button.callback('📢 Telegram', 'out:tg'),
    ],
    [Markup.button.callback('📋 Скопировать', 'out:copy')],
    [Markup.button.callback('🔄 Новая статья', 'restart')],
  ])
}
